BUILD.md

```diff
@@ -0,0 +1,170 @@
+# Build FreeOrion
+
+
+## Hardware and OS Requirements
+
+FreeOrion will compile on most recent Windows, Mac OS X and Linux operating
+systems. Other operating systems have reported to work, but support is not
+actively maintained by the FreeOrion developers. FreeOrion is developed for x86
+compatible processor architectures, other architectures haven't been tested.
+
+The FreeOrion source code makes heavy use of templates and requires much memory
+to compile ; 4GB RAM or more is recommended. To build FreeOrion, expect up to
+45 minutes on a Intel Core i5 system.
+
+
+## Required Software Dependencies
+
+FreeOrion depends on the following software to build:
+
+  * [Visual Studio] - 2015 for Windows Desktop Update 3 ; Windows only
+  * [Xcode] - 6.4 or later ; Mac OS X only
+  * [CMake] - 3.4 (Mac OS X) ; 3.1 or later (Other non-Windows)
+  * A C++11 compliant compiler - Other Operating Systems
+    * [GNU GCC] - 5.0 or later
+    * [Clang] - 3.9 or later
+  * [Python] - 2.7 or later, but not 3 or later
+  * [Git]
+
+FreeOrion depends on the following libraries or APIs to run:
+
+  * OpenGL - 2.1 or later ; usually provided by the graphic card driver or
+    Operating System
+  * OpenGL Utilities (GLU) ; usually provided by the graphic card driver or
+    Operating System
+  * OpenAL - It's recommended to use the [OpenAL Soft] implementation
+  * [Boost] - 1.56 or later
+  * [zlib]
+  * [libpython] - 2.7 or later, but not 3 or later
+  * [FreeType2]
+  * [libpng]
+  * [libogg]
+  * [libvorbis]
+  * [SDL2]
+
+
+## Obtaining FreeOrion Source Code and Software Dependencies
+
+For Windows and Mac OS X a [FreeOrion Software Development Kit] is provided as
+download to compile FreeOrion from source. It contains the preconfigured and
+-compiled build and runtime dependencies for Visual Studio v140 toolchain on
+Windows and Mac OS X 10.9 SDK with Xcode or later on Mac OS X.
```

Comment:
"...for *the* Visual Studio v140 toolchain on Windows and Xcode 6.4 (and later) on Mac OS X."

Or, if you want to mention the required OS versions here too: "...for *the* Visual Studio v140 toolchain on Windows 7 (or later) and Xcode 6.4 (or later) on Mac OS X 10.10 (or later)."

Reply:
Fixed, but I took the first suggestion without the OS version reiteration.